content/docs/reference/config/installation-options/index.md

```diff
@@ -0,0 +1,250 @@
+| `servicegraph.service.name` | `http` |  |
+| `servicegraph.service.type` | `ClusterIP` |  |
+| `servicegraph.service.externalPort` | `8088` |  |
+| `servicegraph.service.internalPort` | `8088` |  |
+| `servicegraph.ingress` | `servicegraph.local` |  |
+| `servicegraph.ingress.enabled` | `false` |  |
+| `servicegraph.ingress.hosts` | `servicegraph.local` |  |
+| `servicegraph.ingress.annotations` |  |  |
+| `servicegraph.ingress.tls` |  |  |
+| `servicegraph.prometheusAddr` | `http://prometheus:9090` |  |
+| `zipkin` |  |  |
+| `zipkin.enabled` | `true` |  |
+| `tracing` | `zipkin.local` |  |
+| `tracing.enabled` | `false` |  |
+| `tracing.jaeger` |  |  |
+| `tracing.jaeger.enabled` | `false` |  |
+| `tracing.jaeger.memory` |  |  |
+| `tracing.jaeger.memory.max_traces` | `50000` |  |
+| `tracing.replicaCount` | `1` |  |
+| `tracing.image` |  |  |
+| `tracing.image.repository` | `jaegertracing/all-in-one` |  |
+| `tracing.image.tag` | `1.5` |  |
+| `tracing.service` |  |  |
+| `tracing.service.annotations` | `{}` |  |
+| `tracing.service.name` | `http` |  |
+| `tracing.service.type` | `ClusterIP` |  |
+| `tracing.service.externalPort` | `9411` |  |
+| `tracing.service.internalPort` | `9411` |  |
+| `tracing.service.uiPort` | `16686` |  |
+| `tracing.ingress` | `zipkin.local` |  |
+| `tracing.ingress.enabled` | `false` |  |
+| `tracing.ingress.hosts` | `zipkin.local` |  |
+| `tracing.ingress.annotations` |  |  |
+| `tracing.ingress.tls` |  |  |
+| `kiali` |  |  |
+| `kiali.enabled` | `false` |  |
+| `kiali.replicaCount` | `1` |  |
+| `kiali.image` |  |  |
+| `kiali.image.repository` | `kiali/kiali` |  |
+| `kiali.image.tag` | `v0.4.0` |  |
+| `kiali.ingress` |  |  |
+| `kiali.ingress.enabled` | `false` |  |
+| `kiali.ingress.annotations` |  |  |
+| `kiali.ingress.tls` |  |  |
+| `kiali.dashboard` |  |  |
+| `kiali.dashboard.username` | `admin` |  |
+| `kiali.dashboard.password` | `admin` |  |
+<!-- AUTO-GENERATED-END -->
+
+The Helm chart also offers significant customization options per individual
+service. Customize these per-service options at your own risk. The per-service options are exposed via
+the [`values.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) file.
+
+## Example: traffic management minimal set
+
+Istio has a rich feature set, but you may only want to use a subset of these. For instance, you might be only interested in installing the minimum necessary to support traffic management functionality.
+
+This example shows how to install the minimal set of components necessary to use [traffic management](/docs/tasks/traffic-management/) features.
+
+Execute the following command to install the Pilot and Citadel:
+
+{{< text bash >}}
+$ helm install install/kubernetes/helm/istio --name istio --namespace istio-system \
+  --set ingress.enabled=false,gateways.istio-egressgateway.enabled=false,galley.enabled=false \
+  --set gateways.istio-ingressgateway.enabled=false,sidecarInjectorWebhook.enabled=false \
```

Comment:
You set ingress gateway to false. However, bookinfo requires gateway.

Reply:
Like @geeknoid commented in a previous review if ingress gateway is there then it isn't really a minimal set. Two options:
1. Remove the "installing the sample Bookinfo application or" part of the sentence
1. Add ingress gateway and change the title to Bookinfo minimal set

Reply:
Would you consider instead of calling this the "minimal set" - "Running Istio without Mixer"?  For performance reasons, folks may not want to run with Mixer as it has a very large impact on QPS.

Another option is to have a minimal set and then add the "run without mixer"

Also, please use the syntax:
helm install install/kubernetes/helm/istio --name istio --namespace istio-system \
 --set ingress.enabled=false
--set gateways.istioegressgateway.enabled=false
--set galley.enabled=false \
--set gateways.istio-ingressgateway.enabled=false \
--set sidecarInjectorWebhook.enabled=false

This will render in a more readable format.